README.md

```diff
@@ -3,53 +3,73 @@ dojos
 
 This is where the Novoda team do all their hacking
 
-# Practicing a new dojo
+# Running a dojo
 
-- find the dojo's module you want to attempt this will be a module under the root dir
-- inside this copy the `default` directory and name it with the date and your name or pairing name
-- this dir you just copied should have instructions to run the dojo as well as any example code
+Find the module of the dojo you want to attempt. This will be a module under the root dir. Inside you will either find the  `default`  directory or a `java` and `swift` directory.
+
+If you only find the  `default` directory, follow [these instructions](https://github.com/novoda/dojos#Updating-a-dojo). Once the folder structure has been updated properly you can continue, make a copy of the default project of whichever language you want to run the kata with.
+
+Go inside this copy of the `default` directory,  and name it with the date and your name or pairing name. This dir you just copied should have instructions to run the dojo as well as any example code.
 
 ```
 -dojos
 |- theKata
-  |- default
-  |- 01012014-yourname-pairsname
+    |- yourChoosenLanguage
+        |- default
+        |- 01012014-yourname-pairsname
 ```
 
-- add the copied kata to the root `settings.gradle` as `theKata:01012014-yourname-pairsname`
+## Android katas
 
+- add the copied kata to the root `settings.gradle` as `theKata:01012014-yourname-pairsname`
 ```
 include: 'theKata:default', 'theKata:01012014-yourname-pairsname'
 ```
 
-# Running a new dojo
+# Starting a new dojo
 
 - create a new directory inside root with the dojo's name
-- create a `default` directory within which contains the build.gradle file and any template source code to start from
-- a dojo level `build.gradle` containing 
-
-```groovy
-subprojects {
-	apply from: '../default/build.gradle'
-}
-```
-should be added to keep all the implementations using the same build
-
+- create two directories inside the dojo's named `swift` and `java`
```

Comment:
What about dojos that are not language dependant at all? Actually we have tons of these. They also do not come with initial source code.
So should we say something like this?

> If the dojo targets specific programming languages like `swift` or `java`, create a directory with the name of the language.